formatTest/typeCheckedTests/expected_output/oo.re

```diff
@@ -30,28 +29,27 @@ let tmp = {
 /**
  * Comment on stackWithAttributes.
  */
```

Comment:
This is intentional. 

I just feel the position of "{" doesn't look good in this case:
```
class virtual stackWithAttributes 'a init =>
/* Before class */
{
```